core/src/test/java/bio/terra/pearl/core/service/kit/pepper/LivePepperDSMClientTest.java

```diff
@@ -22,17 +25,23 @@
 import java.util.concurrent.TimeUnit;
 
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.*;
+import static org.hamcrest.Matchers.contains;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.matchesRegex;
+import static org.hamcrest.Matchers.notNullValue;
 import static org.junit.jupiter.api.Assertions.assertThrows;
 import static org.mockito.Mockito.when;
 
 public class LivePepperDSMClientTest extends BaseSpringBootTest {
 
-    // Common test fixture entities
-    private Enrollee enrollee;
-    private KitRequest kitRequest;
-    private PepperKitAddress address;
 
+    @Autowired
+    private EnrolleeFactory enrolleeFactory;
+    @Autowired
+    private KitRequestFactory kitRequestFactory;
+    @Autowired
+    private ObjectMapper objectMapper;
```

Comment:
thanks for moving these to the top -- our general practice is to only have them at the bottom if it's an extremely long list